rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -338,6 +339,73 @@ def test_exec(config, ssh_con):
         log.info("remove user created")
         reusable.remove_user(user_info[0])
 
+    # CEPH-83589550 - Bug 2262650 - [GSS]Missing Objects in RGW S3 bucket while making GET request - 404 not found error
+    elif config.test_ops.get("test_multipart_upload_with_failed_upload_parts", False):
+        log.info(f"Verify multipart upload with failed upload parts in parallel")
```

Comment:
@hmaheswa, can we remove format string here?

Reply:
@anrao19 
done, please review again